docs/Examples.md

```diff
@@ -4,7 +4,15 @@ The [examples/](/examples) folder contains a number of small services which work
 
 ## Prerequisites
 
-Before all services start we should first start all dependencies with `docker-compose`.
+You will have to have the following dependencies installed in your system to be able to run this examples in your system:
```

Comment:
How about *"To run these examples on your system, you need the following dependencies installed:"*. 

Just avoiding the duplicate 'on your system'.